launch/exec_d_windows.go

```diff
@@ -0,0 +1,22 @@
+package launch
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+	"syscall"
+)
+
+const EnvExecDHandle = "CNB_EXEC_D_HANDLE"
+
+func setHandle(cmd *exec.Cmd, pw *os.File) error {
+	handle := pw.Fd()
+	if err := syscall.SetHandleInformation(syscall.Handle(handle), syscall.HANDLE_FLAG_INHERIT, 1); err != nil {
```

Comment:
I've been using the `golang.org/x/sys/windows` package for all my syscalls these days. They're a mostly just more user-friendly wrappers, though sometimes have better error checking and more defined constants than `syscall`.

Also, it appears that third argument is the correct value but should probably use the same constant instead ([MSFT docs](https://docs.microsoft.com/en-us/windows/win32/api/handleapi/nf-handleapi-sethandleinformation#:~:text=A%20mask%20that%20specifies)):
> ...
> `dwMask`
> A mask that specifies the bit flags to be changed. **Use the same constants shown in the description of dwFlags**.
> 
> `dwFlags`
> Set of bit flags that specifies properties of the object handle. This parameter can be 0 or one or more of the following values.
> | Value | Meaning |
> |---|---|
> | HANDLE_FLAG_INHERIT 0x00000001 | If this flag is set, a child process created with the bInheritHandles parameter of CreateProcess set to TRUE will inherit the object handle.
> |...| |
 
So the call might look like:
```
windows.SetHandleInformation(windows.Handle(handle), windows.HANDLE_FLAG_INHERIT, windows.HANDLE_FLAG_INHERIT)
```